app/bin/tools/remove_package.dart

```diff
@@ -120,6 +119,10 @@ Future removePackage(String packageName) async {
 
     print('Package "$packageName" got successfully removed.');
     print('WARNING: Please remember to clear the AppEngine memcache!');
+
+    print('Committing changes to DB ...');
+    T.queueMutations(deletes: deletes);
+    await T.commit();
```

Comment:
Shouldn't we do this before we claim the package has been successfully removed?

Reply:
Good point, moving the `print` out after the transaction commits.